test/precompiled/precompiled_test.h

```diff
@@ -2,7 +2,5 @@
 
 #include "precompiled/precompiled.h"
 
-#include "test/test_common/printers.h"
```

Comment:
I see, it's part of https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#teaching-google-test-how-to-print-your-values, will fix.